src/apps/answers/filters.py

```diff
@@ -64,3 +69,19 @@ class AppletMultiinformantAssessmentParams(InternalModel):
     target_subject_id: uuid.UUID | None
     source_subject_id: uuid.UUID | None
     activity_or_flow_id: uuid.UUID | None
+
+    @root_validator
```

Comment:
Just remove this ` | None` above and let pydantic validate it

Reply:
Resolve on last commit, Thanks Rodrigo